src/main/java/org/folio/innreach/domain/entity/JobExecutionStatus.java

```diff
@@ -0,0 +1,47 @@
+package org.folio.innreach.domain.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.folio.innreach.domain.entity.base.Auditable;
+
+import java.util.UUID;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@Entity
+@Table(name = "job_execution_status")
+@ToString
+@EqualsAndHashCode(of = "id", callSuper = false)
+public class JobExecutionStatus extends Auditable {
```

Comment:
I wonder should this be called something like `InstanceContributionStatus` or maybe just `ContributionStatus` since there are many instances/contributions being contributed for one job id I think.